docs/config/config.xml

```diff
@@ -26,6 +26,11 @@
     </database>
 
     <CouchDB>
+        <dbName>%COUCH_DATABASE%</dbName>
+        <hostname>%COUCH_HOSTNAME%</hostname>
+        <port>%COUCH_PORT%</port>
+        <admin>%COUCH_USERNAME%</admin>
+        <adminpass>%COUCH_PASSWORD%</adminpass>
```

Comment:
Sorry I just saw this now after you've already made this change.. but why does this need to be in the XML at all?

Changing it can't break LORIS, the config module seems like a reasonable place for it (which would make it easier to provide a patch for existing projects.)